src/main/java/com/ably/kafka/connect/config/ConfigValueEvaluator.java

```diff
@@ -16,13 +35,15 @@ public class ConfigValueEvaluator {
      * #{key} will be replaced with {@code record.key()} if {@code record.key()} is UTF-8 encoded.
      * #{topic} will be replaced with {@code record.topic()}
      *
-     * @param record  The SinkRecord to map
-     * @param pattern The pattern to map
+     * @param record    The SinkRecord to map
```

Comment:
This sort of Git noise, being a line unrelated to the change you're making being changed, is why I avoid attempting to line up 'table layouts' in source code. A single space after the param names is all that's needed.

Reply:
1726377c47596d0033c5dd4728e8c7b880d372dd